### ISSUE TITLE
Fixes for InfoPopover - Styling

### ISSUE THREAD

**Issue by Karol-colony** [JoinColony/colonyDapp#1792]
Currently, styling of info popover look like this:
![image](https://user-images.githubusercontent.com/20819402/64093882-374c2a80-cd5a-11e9-91a4-b7a0d6b429f8.png)

What needs improvement is:
- padding/margins (15px) 
- line height (18px)
- font styling (IBM Plex Mono for wallet address)
- ENS username color as mention here https://github.com/JoinColony/colonyDapp/issues/1787

Please check how it looks on prototype https://www.figma.com/file/IsTzagr0TvzEMf3nwIcTthLm/Task-Workflow?node-id=4448%3A677
![image](https://user-images.githubusercontent.com/20819402/64093943-6cf11380-cd5a-11e9-9461-bd0d662e0e1c.png)

Suggestion: please, open up "code" tab in figma, double check if you are logged in with your figma account, just click any element to preview its styling. If styling exceeds SP of a task, please create separate styling issues for the future.

**Comment by rdig** [comment]
This overlaps partly with #1787, so I'm going to close that one in favor of this

**Comment by collinvine** [comment]
Let's also add this to the avatar that appears on the Admin table in the `/admin` page in Colony Settings.

![Screen Shot 2019-09-02 at 10 38 13 AM](https://user-images.githubusercontent.com/15755488/64128891-c878cc80-cd6d-11e9-86b2-19cdad752e8e.png)

**Comment by rdig** [comment]
Also, this issue needs to fix, just displaying the user's address, if no other data is available: https://github.com/JoinColony/colonyDapp/issues/1784#issue-487513385

**Comment by Karol-colony** [comment]
@collinvine  we did not want to add info popover in admin table, as it would duplicate the information such as wallet address and username which are explictly displayed there. It is redundant there.

**Comment by collinvine** [comment]
@Karol-colony I disagree. 

1. There is no way to copy the user's wallet address
2. There is no way to navigate to their profile from here

We either need to solve those two issues, or simple add the popover here as well.

**Comment by Karol-colony** [comment]
Hm, to be honest I am confused now because we made different call during either design sync or asana revision. We listed out the exceptions and admin panel was one of them as it duplicates the information which is obviously displayed in front of user eyes. I find this a wrong pattern in UI design if we need to display the same information twice. As for the point 1, this should be changed with DLP. We added possibility to show button to copy wallet address on hover. For the point 2, adding popover only to go to user profile would be there not a slick solution. Clicking avatar could open the profile, that would be my expectation.
